config/access-policy.json

```diff
@@ -106,7 +106,9 @@
         "imagebuilder:GetInfrastructureConfiguration",
         "imagebuilder:TagResource",
         "imagebuilder:UntagResource",
-        "cognito-idp:DeleteUserPool"
+        "cognito-idp:DeleteUserPool",
+        "codebuild:DeleteProject",
+        "route53:*"
```

Comment:
There was a iam:ListPolicyVersion permission needed right?

Reply:
The policy was already there on line no. 48 that's why didn't added that.